src/microbe_stage/membrane_system.cpp

```diff
@@ -288,17 +288,15 @@ void
 
 
     // Set the membrane material //
-    if(!coloredMaterial) {
-        auto baseMaterial = chooseMaterialByType();
+    auto baseMaterial = chooseMaterialByType();
```

Comment:
If you don't change this to be only ran when needed, could you add a todo comment like:
`// TODO: the material needs to be only recreated when the species properties change, not every time an organelle is added or removed`